src/components/Application.tsx

```diff
@@ -551,10 +573,41 @@ export class Application extends React.Component<Props, State> {
     ])
   }
 
+  //
+  // Inactivity Timeout
+  //
+
+  // Increment the idle time counter every minute.
+  private startIdleTimer() {
+    this.setState({
+      idleTime: 0,
+    })
+
+    this.state.idleInterval = setInterval(this.timerIncrement, 60000)
+    return null
+  }
+
+  private timerIncrement() {
+    if (this.state.isLoggedIn && !this.state.isSessionExpired) {
+      this.setState({
+        idleTime: this.state.idleTime + 1,
+      })
+      console.log('idleTimer at ' + this.state.idleTime.toString())
+      if (this.state.idleTime > 14) {
```

Comment:
Can you pull this number into `config.ts` as `SESSION_IDLE_TIMEOUT` or something similar?

Avoids having magic numbers scattered through the codebase.